build/deplicense.sh

```diff
@@ -43,13 +43,19 @@ function isLGPLV3() {
   grep -q '[[:space:]]Version 3' "$1"
 }
 
+function isMIT() {
+  if grep -q 'MIT License' "$1"; then
+    return 0
+  fi
+  grep -q 'Permission is hereby granted, free of charge, to any person' "$1" && \
+  grep -q 'The above copyright notice and this permission notice' "$1"
+}
+
 function inspect() {
   local dir="$1"
 
-  local files="${toplevel}/LICENSE"
-  files="${files} ${toplevel}/COPYING"
-  files="${files} ${toplevel}/internal/LICENSE"
-  files="${files} ${toplevel}/internal/COPYING"
+  local files="$(ls ${toplevel}/{LICENSE,COPYING} 2>/dev/null)"
```

Comment:
My bash hygiene spidey senses tingle at the sight of `ls` in shell scripting. This looks like it should be some invocation of `find "${toplevel}" -type f -iname '....' -o <...>`